src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj

```diff
@@ -25,25 +25,21 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Uno.WinUI" Version="4.4.13" />
+		<PackageReference Include="Uno.WinUI" Version="5.0.19" />
 	</ItemGroup>
 
 	<ItemGroup>
 		<!--Microsoft.SourceLink.GitHub is needed for Source Link support -->
 		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
 	</ItemGroup>
 
-	<PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
```

Comment:
Don't remove WASM support. Keep this but use the `net7.0` tfm for the condition.